From 4f33ce37ab3a6bb11dc29bc345cae0d3885477b6 Mon Sep 17 00:00:00 2001
From: Yura <y.abrahamyan@codeex.io>
Date: Mon, 16 Oct 2023 23:33:25 +0400
Subject: [PATCH 2/3] updated firstFile

---
 firstFile.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/firstFile.txt b/firstFile.txt
index fa2da6e..ac8f01d 100644
--- a/firstFile.txt
+++ b/firstFile.txt
@@ -1,9 +1,9 @@
 line 1
-line 2
+modified line 2
 line 3
 line 4
 line 5
-line 6
+modified line 6
 line 7
 line 8
 line 9
-- 
2.34.1

